config/database.yml

```diff
@@ -20,10 +20,14 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  host: <%= ENV.fetch("DB_HOST", "localhost") %>
+  username: <%= ENV.fetch("DB_USERNAME", "postgres") %>
+  port: <%= ENV.fetch("DB_PORT", "5432") %>
```

Comment:
If you already have a local version of postgres, it'll very likely hog the default port of 5432. It may be easier for developers to simply run the dockerised database on another port rather than delete their local postgres service.